### ISSUE TITLE
Site support for AlfaTV

### ISSUE THREAD

**Issue by Jiriss** [yt-dlp/yt-dlp#1410]
### Checklist

- [X] I'm reporting a new site support request
- [X] I've verified that I'm running yt-dlp version **2021.10.22**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that none of provided URLs [violate any copyrights](https://github.com/ytdl-org/youtube-dl#can-you-add-support-for-this-anime-video-site-or-site-which-shows-current-movies-for-free) or contain any [DRM](https://en.wikipedia.org/wiki/Digital_rights_management) to the best of my knowledge
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [ ] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and am willing to share it if required

### Example URLs

- Single video: https://www.permanto.fi/en/web/alfatv/player/vod?assetId=129355418

### Description

Please could you add support for finnish TV channel AlfaTV? Their videos are located under the domain permanto.fi. This is a example video: https://www.permanto.fi/en/web/alfatv/player/vod?assetId=129355418

### Verbose log

```shell
$ yt-dlp --version
2021.10.22
$ yt-dlp -F https://www.permanto.fi/fi/web/alfatv/player/vod?assetId=129355418
[generic] vod?assetId=129355418: Requesting header
WARNING: [generic] Falling back on generic information extractor.
[generic] vod?assetId=129355418: Downloading webpage
[generic] vod?assetId=129355418: Extracting information
WARNING: [generic] vod?assetId=129355418: Failed to parse JSON Invalid control character at: line 6 column 551 (char 655)
ERROR: [generic] Unsupported URL: https://www.permanto.fi/fi/web/alfatv/player/vod?assetId=129355418
```

**Comment by HobbyistDev** [comment]
It works for version `2020.05.18`. Here is the log:
```
C:\Users\USER\Downloads\>yt-dlp -sv "https://www.permanto.fi/en/web/alfatv/player/vod?assetId=129355418"
[debug] Command-line config: ['-sv', 'https://www.permanto.fi/en/web/alfatv/player/vod?assetId=129355418']
[debug] Portable config "C:\Users\USER\Downloads\yt-dlp\yt-dlp.conf": ['-v', '--netrc-location', 'C:UsersUSERDownloadsyt-dlp.netrc', '-f', 'bv+ba/b', '--compat-options', 'format-sort', '--embed-metadata', '--embed-subs', '--sub-langs', 'all', '--console-title', '--downloader', 'aria2c']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.05.18 [b14d523] (win_exe)
[debug] Compatibility options: format-sort
[debug] Python version 3.8.10 (CPython 64bit) - Windows-10-10.0.19044-SP0
[debug] Checking exe version: ffprobe -bsfs
[debug] Checking exe version: ffmpeg -bsfs
[debug] exe versions: ffmpeg 5.0.1-full_build-www.gyan.dev (setts), ffprobe 5.0.1-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.14.1, brotli-1.0.9, certifi-2021.10.08, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] [Icareus] Extracting URL: https://www.permanto.fi/en/web/alfatv/player/vod?assetId=129355418
[Icareus] 129355418: Downloading webpage
[Icareus] 129355418: Downloading JSON metadata
[Icareus] 129355418: Downloading m3u8 information
[debug] Sort order given by user: hasaud, lang, quality, tbr, filesize, vbr, height, width, proto, vext, abr, aext, fps, fs_approx, source, id
[debug] Formats sorted by: hasvid, ie_pref, hasaud, lang, quality, tbr, filesize, vbr, height, width, proto, vext, abr, aext, fps, fs_approx, source, id, res, hdr:12(7), vcodec:vp9.2(10), acodec, asr
[info] 129355418: Downloading 1 format(s): hls-2728
```

**Comment by pukkandan** [comment]
Fixed by #3320

**Comment by Jiriss** [comment]
@pukkandan This TV channel has gone bankrupt in 2022 and therefore there is no reason to continue to support it. Source: https://yle.fi/a/3-12681313
